mappings/net/minecraft/entity/mob/HoglinBrain.mapping

```diff
@@ -0,0 +1,71 @@
+CLASS net/minecraft/class_4835 net/minecraft/entity/mob/HoglinBrain
+	FIELD field_22369 AVOID_MEMORY_DURATION Lnet/minecraft/class_4801;
+	METHOD method_24662 makeRandomWalkTask (F)Lnet/minecraft/class_4118;
+		ARG 0 speed
+	METHOD method_24664 refreshActivities (Lnet/minecraft/class_4760;)V
+		ARG 0 hoglin
+	METHOD method_24665 onAttacking (Lnet/minecraft/class_4760;Lnet/minecraft/class_1309;)V
+		ARG 0 hoglin
+		ARG 1 target
+	METHOD method_24666 addCoreTasks (Lnet/minecraft/class_4760;Lnet/minecraft/class_4095;)V
+		ARG 0 hoglin
+		ARG 1 brain
+	METHOD method_24668 create (Lnet/minecraft/class_4760;Lcom/mojang/datafixers/Dynamic;)Lnet/minecraft/class_4095;
+		ARG 0 hoglin
+		ARG 1 data
+	METHOD method_24669 isWarpedFungiAround (Lnet/minecraft/class_4760;Lnet/minecraft/class_2338;)Z
+		ARG 0 hoglin
+		ARG 1 pos
+	METHOD method_24671 playSoundAtChance (Lnet/minecraft/class_4760;)V
+		ARG 0 hoglin
+	METHOD method_24672 onAttacked (Lnet/minecraft/class_4760;Lnet/minecraft/class_1309;)V
+		ARG 0 hoglin
+		ARG 1 attacker
+	METHOD method_24673 addIdleTasks (Lnet/minecraft/class_4760;Lnet/minecraft/class_4095;)V
+		ARG 0 hoglin
+		ARG 1 brain
+	METHOD method_24674 getMovementSpeed (Lnet/minecraft/class_4760;)F
+		ARG 0 hoglin
+	METHOD method_24675 askAdultsToAvoid (Lnet/minecraft/class_4760;Lnet/minecraft/class_1309;)V
+		ARG 0 hoglin
+		ARG 1 target
+	METHOD method_24676 addFightTasks (Lnet/minecraft/class_4760;Lnet/minecraft/class_4095;)V
+		ARG 0 hoglin
+		ARG 1 brain
+	METHOD method_24677 isNearPlayer (Lnet/minecraft/class_4760;)Z
+		ARG 0 hoglin
+	METHOD method_24678 avoidEnemy (Lnet/minecraft/class_4760;Lnet/minecraft/class_1309;)V
+		ARG 0 hoglin
+		ARG 1 target
+	METHOD method_24679 addAvoidTasks (Lnet/minecraft/class_4760;Lnet/minecraft/class_4095;)V
+		ARG 0 hoglin
+		ARG 1 brain
+	METHOD method_24680 isPacified (Lnet/minecraft/class_4760;)Z
+		ARG 0 hoglin
+	METHOD method_24681 avoid (Lnet/minecraft/class_4760;Lnet/minecraft/class_1309;)V
+		ARG 0 hoglin
+		ARG 1 target
+	METHOD method_24682 hasIdleActivity (Lnet/minecraft/class_4760;)Z
+		ARG 0 hoglin
+	METHOD method_24683 targetEnemy (Lnet/minecraft/class_4760;Lnet/minecraft/class_1309;)V
+		ARG 0 hoglin
+		ARG 1 target
+	METHOD method_24684 getNearestVisibleTargetablePlayer (Lnet/minecraft/class_4760;)Ljava/util/Optional;
+		ARG 0 hoglin
+	METHOD method_24685 setAttackTarget (Lnet/minecraft/class_4760;Lnet/minecraft/class_1309;)V
+		ARG 0 hoglin
+		ARG 1 target
+	METHOD method_24686 hasMoreHoglinsAround (Lnet/minecraft/class_4760;)Z
```

Comment:
`has` -> `are`?

Reply:
I would do `are` if the method triggers the logic to look around, and `has` if it's just returning a boolean from somewhere

Reply:
This checks if hoglins outnumber piglins, and will make hoglins fight back if true